sessionctx/stmtctx/stmtctx.go

```diff
@@ -131,7 +131,7 @@ type StatementContext struct {
 	MemTracker       *memory.Tracker
 	RuntimeStatsColl *execdetails.RuntimeStatsColl
 	TableIDs         []int64
```

Comment:
1. How about using table names as well?
2. How about using the `table_name:index_name` format to display table and index names?

Reply:
tableIDs is not be printed in the slow log.

Reply:
If we do not print the table id, how do we know which table is accessed with an index if there are multiple tables doing the join operation?

Reply:
How about store `table_name:index_name` in indexNames?

Reply:
Sounds reasonable.

Reply:
done.